opencog/cython/opencog/atomspace_details.pyx

```diff
@@ -411,8 +435,10 @@ cdef api object py_atomspace(cAtomSpace *c_atomspace) with gil:
     cdef AtomSpace atomspace = AtomSpace_factory(c_atomspace)
     return atomspace
 
-cdef api object py_atom(UUID uuid, object atomspace):
-    cdef Handle temphandle = Handle(uuid)
-    cdef Atom atom = Atom(temphandle, atomspace)
+cdef api object py_atom(void* atom_ptr, object atomspace):
```

Comment:
The api for this can take any type you want, as far as I can tell. So yes, you could pass a cHandle to this function, by value. But that won't help below.